DuckDuckGo/OnboardDefaultBrowserViewController.swift

```diff
@@ -19,7 +19,7 @@
 import UIKit
 import Core
 
-class OnboardingDefaultBroswerViewController: OnboardingContentViewController {
+class OnboardDefaultBrowserViewController: OnboardingContentViewController {
```

Comment:
Good spotting on the `Browser` type – however we should keep the "Onboarding" part of the title. 🙂 

```suggestion
class OnboardingDefaultBrowserViewController: OnboardingContentViewController {
```